src/main/utils.ts

```diff
@@ -133,3 +137,34 @@ export const openFilePicker = (): Promise<
     }
   });
 };
+
+export const cleanup = async () => {
+  const captainProcess = global.captainProcess as ChildProcess;
+  log.info(
+    "Cleanup function invoked, is captain running? ",
+    !(captainProcess?.killed ?? true),
+  );
+  if (captainProcess && captainProcess.exitCode === null) {
+    const success = killCaptain();
+    if (success) {
+      global.captainProcess = null;
+      log.info("Successfully terminated captain :)");
+    } else {
+      log.error("Something went wrong when terminating captain!");
+    }
+  }
+};
+
+export const loadFileFromFullPath = async (
+  filePath: string,
+): Promise<string> => {
+  return fs.readFileSync(filePath, { encoding: "utf-8" }).toString();
+};
+
+export const saveFileToFullPath = async (
+  filePath: string,
+  content: string,
+): Promise<Result<void>> => {
+  fs.writeFileSync(filePath, content);
```

Comment:
you need to try/catch this as well, the only result your function is ever going to return will be Ok(()) but it can still throw an error as usual
```typescript
try {
  fs.writeFileSync(filePath, content);
  return Ok()
}
catch (e) {
  return Err(e);
}